### PR TITLE
Updates to setup_nano for kernel 4.6.2

### PR DESCRIPTION
**Pull request by itsarune**
### Description

The new kernel running on the Jetsons requires a slightly different device tree

### Testing Done

- Used at robocup

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

**Review by nimazareian — DISMISSED** [review]
I'm not familiar with Ansible or the device tree, so I will leave the more detailed review to @LiCody and @PhilD71

**Review by PhilD71 — COMMENTED** [review]
It looks good but im unfamiliar with ansible as well so maybe Cody should look at it

File: src/software/jetson_nano/ansible/playbooks/setup_nano.yml [hunk 1]
```diff
@@ -141,6 +141,17 @@
           args:
             chdir: ~/
 
+        - name: Load spidev on boot
```
Comment (LiCody) [inline src/software/jetson_nano/ansible/playbooks/setup_nano.yml]: This seems sketchy if setup_nano is run multiple times. From quick search unknown if duplicates mess things up

**Review by JoshuaWu7 — DISMISSED** [review]

**Review by JoshuaWu7 — COMMENTED** [review]
Looks good to me, pending Cody's comment

**Comment by nimazareian** [comment]
@itsarune Pending Cody's comment on this PR.

File: src/software/jetson_nano/ansible/playbooks/setup_nano.yml (hunk 1 above)
Comment (itsarune) [reply src/software/jetson_nano/ansible/playbooks/setup_nano.yml]: fixed this because I added it to the kernel iso so we don't need to add it ourselves

**Review by nimazareian — APPROVED** [review]

**Review by nimazareian — APPROVED** [review]

**Review by Bvasilchikov — APPROVED** [review]

**Pull request merged**